src/main/steps/edge-case/full-name/content.test.ts

```diff
@@ -70,9 +63,5 @@ describe('Applicant > full-name', () => {
   test('should contain submit button', () => {
     expect((form.submit.text as Function)(generatePageContent({ language: 'en' }))).toBe('Save and continue');
```

Comment:
1) All the toBe content should be read from translation file.

2) Also create constants for 'en' and 'cy'

const EN = 'en';
const CY = 'cy';